src/schemas/swap_quote_request_schema.json

```diff
@@ -21,6 +21,17 @@
         },
         "excludedSources": {
             "type": "string"
+        },
+        "apiKey": {
+            "type": "string"
+        },
+        "intentOnFilling": {
+            "type": "string",
```

Comment:
Can we use a `boolean` type here? http://json-schema.org/understanding-json-schema/reference/boolean.html

Reply:
I wanted to.  I tried.  `jsonschema` choked on it.

Reply:
At test runtime i get this json schema error

`{\"field\":\"instance.intentOnFilling\",\"code\":1001,\"reason\":\"is not of a type(s) boolean\"}`

This happened no matter whether my client said `quote?intentOnFilling` or `quote?intenOnFilling=true`.

Reply:
Got it.

I think this schema could be cleaner if we:
-  get rid of the `flag` functionality (i.e. setting `?intentOnFilling` indicating that `intentOnFilling` is equal to `true`).  I have not seen this used in query parameters before, and IMHO is not super important to support
- Use an `enum` type to force the consumer of the API to specify the text `true`, `false` for our 'boolean' params (intentOnFilling, skipValidaiton).  (See https://stackoverflow.com/a/16826238 and https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values
   - Make sure to keep specifying these params optional though

This would allow `?skipValidation=truue` to raise a schema error instead of silently being set to `false` on our backend.

Reply:
How's this: https://github.com/0xProject/0x-api/pull/162/commits/bb2814a870d8c874f6f83b4e92d185a25c434ade